### PR TITLE
github-cli: new package at 0.10.1

### PR DESCRIPTION
**Pull request by mmatongo**
---

## Description of package

GitHub’s official command line tool
## Installed manifest of package

```
/var/db/kiss/installed/github-cli/version
/var/db/kiss/installed/github-cli/sources
/var/db/kiss/installed/github-cli/manifest
/var/db/kiss/installed/github-cli/depends
/var/db/kiss/installed/github-cli/checksums
/var/db/kiss/installed/github-cli/build
/var/db/kiss/installed/github-cli/
/var/db/kiss/installed/
/var/db/kiss/
/var/db/
/var/
/usr/share/zsh/site-functions/_gh
/usr/share/zsh/site-functions/
/usr/share/zsh/
/usr/share/licenses/github-cli
/usr/share/licenses/
/usr/share/bash-completion/completions/gh
/usr/share/bash-completion/completions/
/usr/share/bash-completion/
/usr/share/
/usr/bin/gh
/usr/bin/
/usr/
```

## New package

- [Y] Latest upstream version.
- [Y] I agree to maintain this package.
- [Y] I agree to notify @dylanaraps if I can no longer maintain this package.

### CODE REVIEW

**Comment by mmatongo** [comment]
@dylanaraps it builds just fine, that's the recommended command usage.

```
Usage:  gh completion [flags]

Flags:
  -s, --shell string   Shell type: {bash|zsh|fish|powershell}

```

File: community/github-cli/build [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/bin/sh -e
+
+make 
+
+./bin/gh completion -s bash > bash
+./bin/gh completion -s zsh > zsh
```
Comment (dylanaraps) [inline community/github-cli/build:6]: Do we really need to include these? To silence shellcheck just change `> file` to `> ./file`

Comment (mmatongo) [reply community/github-cli/build:6]: Yes and No. If you like the convenience of completion then yes if you can live without it then no, I assumed the former as it is something you can negligibly live with.

**Comment by mmatongo** [comment]
Sorry I had to close it cause it wasn't committing my changes.

**Pull request closed without merging**